src/permission/rule-parser.js

```diff
@@ -0,0 +1,109 @@
+var rulesMap = require( './rules-map' );
+var FUNCTION_REGEXP = /([a-zA-Z0-9_]*)\(/g;
```

Comment:
You could use \w instead:

var FUNCTION_REGEXP = /([\w_]*)\(/g;
var NEW_REGEXP = /^new[^\w^]|[^\w^]new[^\w]/;
var OLD_DATA_REGEXP = /^oldData[^\w^]|[^\w^]oldData[^\w]/;
var DATA_REGEXP = /^data[^\w^]|[^\w^]data[^\w]/;
var CROSS_REFERENCE_REGEXP = /^_[^\w^]|[^\w^]_[^\w]/;